pallets/parachain-staking/src/benchmarks.rs

```diff
@@ -80,6 +84,34 @@ fn create_funded_delegator<T: Config>(
 		0u32,
 		0u32, // first delegation for all calls
 	)?;
+
+	{
```

Comment:
There should be a `TODO` here for replacing this code with the fungible methods once the migration has been complete. This will help reducing the technical debt and anyone in the future to understand why this code was necessary.